src/components/Issue.tsx

```diff
@@ -81,6 +83,18 @@ const Issue = ({
     //console.log("Excluded " + repoName + " - " + title + " - for being a test, tutorial or fake issue");
     return null;
   }
+
+  const isupTimeRepo = () => {
+    // uptime status issues that are automatically generated
+    if (title.endsWith("is down")) {
+      return true;
+    }
+    return false;
+  };
+
+  if(isupTimeRepo() == true && upTime) {
```

Comment:
One little nitpick...could you change this line to 
```
if(upTime && isupTimeRepo())
```

(The  "== true" is unnecessary and I think it would be more efficient to check the upTime flag before calling the function, because if it is false, then it won't call the function). After you make this change, I'll merge it..thanks so much!

Reply:
Oh yes